### DISCUSSION TITLE
Status bar no longer shows sizes of contents

### DISCUSSION THREAD

**Discussion by pavelsko67** [Open-Shell/Open-Shell-Menu#1334] [Q&A]
Unfortunately, I can't get the open shell work with status bar. Status bar only activates by standart windows explorer menu and has no info about the size of the contents, disk memory etc. Even if I tweak anything in openshell explorer settings, nothing changes, I can't activate or deactivate status bar by openshell =(

windows 10

**Comment by pavelsko67** [comment]
Really Nobody faced such problem?

**Comment by tallman69** [comment]
I have never had a problem with that.  What version of Windows are you using?  What version of Open-Shell are you using?

**Reply by pavelsko67** [reply]
![Screen Shot 03-03-23 at 08 01 AM](https://user-images.githubusercontent.com/114736522/222637584-beb13487-deda-4188-9a4c-3d00abc22ab9.PNG)
![Screen Shot 03-03-21 at 08 01 AM](https://user-images.githubusercontent.com/114736522/222637586-7fdb8c01-04cc-48af-8aca-0f037b36b852.png)
windows 10 19044.2604 and latest Openshell
I found, that somehow the openshell doesn't integrate in the windows explorer. 
installed and uninstalled severel times openshell and slassicshell, result the same. However, the tweal whith start menu (aka xp) works fine.

**Comment by tallman69** [comment]
Are you using the latest stable release (4.4.170) of Open-Shell, or the latest pre-release (4.4.189)?
Has the Classic Explorer status bar worked for you in the past?  If so, do you recall if it stopped working after an update or a program was installed on your computer?

**Comment by tallman69** [comment]
One thing you might consider doing is performing a clean boot to see if there is a service or another program that is conflicting with the Classic Explorer status bar setting.  https://support.microsoft.com/en-us/topic/how-to-perform-a-clean-boot-in-windows-da2f9573-6eec-00ad-2f8a-a97a1807f3dd#ID0EBBD=Windows_10

If you find that there is a service that is causing the problem, do not leave it disabled in the System Configuration Manager.  But rather disable it in the Serices Manager which can be accessed by typing services.msc in the run box and then hitting enter..

**Comment by tallman69** [comment]
Hey, pavelsko67, did you ever get the Classic Explorer status bar to work?  If so, how did you get it to work?
